### DISCUSSION TITLE
Request: Fake Delay

### DISCUSSION THREAD

**Discussion by usernotnull** [livewire/livewire#1724] [4. General]
I thought it would be a great feature to add a config option to add delays to the XHR calls.
That way when running locally, we can simulate a live situation better by introducing the lag users usually face after performing an action (such as clicking a button and waiting...)

**Comment by nuernbergerA** [comment]
![image](https://user-images.githubusercontent.com/13331388/94675563-364d4a00-031a-11eb-94eb-f484af21c6e6.png)

**Comment by usernotnull** [comment]
I don't know why I forgot about this :)
Thanks
